README.md

```diff
@@ -1 +1,61 @@
-# capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Designing a Responsive page Using HTML&CSS
+
+
+> Description the project.
+
+
+## Built With
+
+- html&css
+- no Frameworks
+- Flexbox
+- linter
+
+## Link to online version of my capstone
+
+https://github.com/ilyassatlassi/capstone
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+All you need VScode compiler or a compiler of your choice.
+
+### Setup
+
+You will need to clone the url from the source repository and run the following command 
+git clone https://github.com/ilyassatlassi/capstone.git
+
+
+## Outline
+
+- [@demo](https://www.loom.com/share/c9d1cdea8e4141ac81eecb4c591c5987)
+## Author
+
+👤 ilyass atlassi
+
+- GitHub: [@githubhandle](https://github.com/ilyassatlass)
+- Twitter: [@twitterhandle](https://twitter.com/ilyass_atlassi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ilyassatlassi/)
+
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] Please make sure that your  README has an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) :smile: